package.json

```diff
@@ -24,30 +24,31 @@
   },
   "dependencies": {
     "@socket.io/redis-adapter": "^8.3.0",
-    "@tsed/ajv": "^8.3.5",
-    "@tsed/common": "^8.3.5",
-    "@tsed/core": "^8.3.5",
-    "@tsed/di": "^8.3.5",
-    "@tsed/engines": "^8.3.5",
-    "@tsed/exceptions": "^8.3.5",
-    "@tsed/hooks": "^8.3.5",
-    "@tsed/json-mapper": "^8.3.5",
+    "@tsed/ajv": "^8.4.0",
```

Comment:
same things. it should be extracted to another technical PR.

Reply:
normally, i'd agree, but this PR has scope creep, it's more than just add album feature, it includes all the transfomer work, etc.. so i will just shove thin this PR. along with the eslint changes. cos i am lazy LMAO 💯